doc/en/components/charts/chart-types-area.md

```diff
@@ -30,7 +36,7 @@ mentionedTypes: ["XamCategoryChart", "XamDataChart"]
 
 ## $PlatformShort$ Advanced Area Chart
 
-If you need an area chart with more features such as composite other series, you can use DataChart component.
+If you need an Area Chart with more features such as composite other series, you can configure the marker outlines, marker brushes, series brushes, and series outlines as demonstrated below.
 
```

Comment:
this sentence doesn't make sense, If I need an area chart with more features, I just configure?

Reply:
Changed to 

"Additional styling settings can be configured on series such marker outlines, marker brushes, series brushes, and series outlines as demonstrated below."